File: PHP/car.php
<?php
require_once('account.php');
class Car {
    public $id;
    public $license;
    public $driver;
    protected $passengers;

    public function_construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }
    
    public function PrintDataCar() {
        echo "license: $this->license, Driver:".$this->driver->name;
    }

    public function getPassenger() {
        return $this->passenger;
    }

    public function setPassenger($passenger) {
        
        if (passenger == 4) {
            $this->passenger = $passenger;
        }
        else {
            echo "Necesitas asignar 4 pasajeros";
        }
    }

    public function printDataCar() {
        echo "
        Licencia: $this->license
        Driver: {$this->driver->name}
        Número de pasajeros: $this->passenger
        ";
    }
}
?>

File: Java/Car.java
class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer passegenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
        
    }

    void printDataCar() {
        if(passegenger != null){
            System.out.println("License: " + license + " Name Driver: " + driver.name + "Passengers: " + passegenger);    
        }
        g
    }

    public Integer getPassenger(){
        return passegenger;
    }

    public void setPassenger(Integer passenger) {
        if(passenger == 4){
            this.passegenger = passenger;
        }else {
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }

    

}